pkg/clusterautoscaler/flags.go

```diff
@@ -153,7 +160,7 @@ func AddClusterAutoscalerFlags(cmd *cobra.Command, prefix string) *AutoscalerArg
 	cmd.Flags().IntVar(
 		&args.ResourceLimits.MaxNodesTotal,
 		fmt.Sprintf("%s%s", prefix, maxNodesTotalFlag),
-		249,
+		0,
```

Comment:
Should default be 0? Maybe nil and use pointer to int instead?

Reply:
Setting 0 seems to omit the "default" suggestion in the "-help" option screen in the CLI.
Since this value tends to change depending on multiple variable conditions, i've felt that 0 might be the best substitution to hide the default suggestion, as well as prevent any "nil" ptr exceptions.

Reply:
Oh I see